chingsley/ireporter
# Ft fetch deleterecord #163060628

#### What does this PR do?
This pull request deals with implementation of fetch to delete a record.

#### Description of Task to be completed?
- write the deleteRecord() function recordhistory.js file to implement the fetch
- using fetch consume the API with the route /api/v1/<record-type>/<record-id, setting up the request object to use the 'DELETE' method.
- before deleting the red-flag, show a warning dialog box so that users can confirm whether they really want to perform the delete.
- display error messages by calling the 'showDialogMsg()' function with the right arguments
- on successful delete, call the showDialogMsg() function with the right arguments to show successful message.

#### How should this be manually tested?
- clone the repo
- cd into the root directory
- run `npm install' to install the necessary dependencies
- run `npm test' to see the tests pass
- run `npm start' to start the server
- ensure that you are a registered user as an admin. To delete a record, upon login, ensure you have created at least one record. Click on the report history link to view all records. You can then click on the delete button, which the round button on the top right corner marked 'X'. NOTE: only records of status 'draft' can be deleted. 

#### What are the relevant pivotal tracker stories
 #163060628
